LinkedListProject/SingleLinkedList.cpp

```diff
@@ -4,4 +4,19 @@
 using namespace std;
 
 SingleLinkedList::SingleLinkedList() : LinkedList() {
+
+}
+
+void SingleLinkedList::InsertFirst(int data) {
+	Node* newNode = new Node(data);
+	newNode->next = GetFirstNode(); //oldFirst -> second 
+	SetFirstNode(newNode); //newLink -> first
+}
+
+void SingleLinkedList::DeleteFirst() {
+	if (IsEmpty()) return;
+	Node* temp = GetFirstNode();
+	SetFirstNode(GetFirstNode()->next);
```

Comment:
ở đây có 1 rule k chặt là: không cần phải dùng get method cho method ở cùng class, nghĩa là cùng class "có thể" dùng this.first thay vi GetFirstNode()

Tương tự với logic, -> next có thể gói vào trong method: FetchNext()